src/pages/MyPage/MyReview.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const MyReview = ({ review, goDelete }) => {
+  const { text, id, date, sitter } = review;
+
+  const reviewMonth = (new Date(date).getMonth() + 1).toString();
+  const reviewDate = new Date(date).getDate().toString();
+  const reviewHour = new Date(date).getHours().toString();
+  const reviewMinute = new Date(date).getMinutes().toString();
+
+  return (
+    <MyReviewWrapper>
+      <MyReviewLeft>
+        <ReviewSitterName>{sitter} 펫시터님</ReviewSitterName>
+        <ReviewContent>{text}</ReviewContent>
+      </MyReviewLeft>
+      <MyReviewRight>
+        <ReviewTime>
+          {reviewMonth}월 {reviewDate}일 {reviewHour}시 {reviewMinute}분
+        </ReviewTime>
+        <ReviewDelete
+          onClick={() => {
+            goDelete(id);
+          }}
```

Comment:
위 리뷰대로라면 단순히 goDelete 함수를 전달해주기만 하면 됩니다.
```suggestion
          onClick={goDelete}
```